### DISCUSSION TITLE
i386 installation

### DISCUSSION THREAD

**Discussion by jimix974** [jasonish/evebox#277] [General]
HI,

I try to install evebox but I have some problem

first I try to install with apt but apt can't find the packet, I try to edit the sourcelist but I have the same error.

I try to install manually but when I download  the package for debian, I receive au file with "amd64" in the name file. (evebox-0.17.2-amd64.deb)
I try to install the deb file but there is an error because the file is for amd64

Could you help me ? thanks a lot

**Comment by jasonish** [comment]
At this time I don't do 32 bit x86 builds.
